Sources/KeyboardObserving/viewmodifier/KeyboardObserving.swift

```diff
@@ -10,13 +10,13 @@ import UIKit
 
 
 struct KeyboardObserving: ViewModifier {
-
+  var padding: CGFloat = 8
```

Comment:
Because of the `edgesIgnoringSafeArea` view modifier, this adds _extra_ space between the keyboard and the bottom of the view.

Is that what you’re intending, or is this fixing a UI bug you’re experiencing?

Reply:
Yes - the bottom of the view is the text field. If there is no space - it looks cramped.